examples/postgres/advanced-blog-cli/src/pagination.rs

```diff
@@ -0,0 +1,69 @@
+use diesel::prelude::*;
+use diesel::query_dsl::methods::LoadQuery;
+use diesel::query_builder::*;
+use diesel::pg::Pg;
+use diesel::types::BigInt;
+
+pub trait Paginate: Sized {
```

Comment:
Not that it is particularly useful here, but in the 'leading by example' sense we should probably add some doc comments to the public items in this module. At least, we should add a module level doc comment that links to the #1437 guide (once it is online)

Reply:
I agree. I'm going to merge without doing this so I can publish a guide referencing code here, but I will make sure to come back and do this after we ship 1.0